bot.rb

```diff
@@ -334,6 +340,37 @@ def process_code_signing(issue)
       return body.join("\n\n")
     end
 
+    def fetch_and_process_pinned_issues
+      logger.info("Fetching pinned issues from '#{SLUG}'...")
+      result = GitHubAPI::Client.query(GitHubAPI::Query::PINNED_ISSUES, variables: { owner: REPOSITORY_OWNER, name: REPOSITORY_NAME })
+      result.data.repository.pinned_issues.edges.each do |edge|
+        pinned_issue = edge.node.issue
+        if pinned_issue.closed
+          slack_about_closed_pinned_issues(pinned_issue)
+        end
+      end
+    end
+
+    def slack_about_closed_pinned_issues(pinned_issue)
+      return unless Bot.should_send_trivial_slack_notification?
+
+      logger.info("Notifying the Slack room about pinned issue that is closed...")
+
+      post_body = {
+        text: "Caution: Item ##{pinned_issue.number} is still a pinned issue although it was recently closed: #{pinned_issue.url}"
+      }.to_json
+
+      logger.info(post_body)
```

Comment:
❓ I assume this was for debugging, could we remove it now? (to avoid logging twice)

Reply:
Absolutely, I am gonna remove it, thanks 👍